tests/acceptance/features/webUIFirstrunwizard/reset_all.feature

```diff
@@ -0,0 +1,26 @@
+@webUI @insulated @disablePreviews
+Feature:  first login wizard
+  As a user
+  I want to see basic features of ownCloud when I first log in to owncloud
+  So that I can get basic overview about ownCloud
+
+  Scenario: Administrator runs reset-all occ command
+    Given user "user1" has been created with default attributes
```

Comment:
We could also create `user2` and not login to `user2` in the given section.
Then also check below that the new `user2` (that is already waiting to see the first run wizard) still sees the popup after the `reset-all` command.
(I can imagine that the `reset-all` command might have a bug where it flips the wizard state for all users, instead of resetting it to "not seen yet")